src/backend/joanie/core/admin.py

```diff
@@ -182,30 +182,30 @@ def get_urls(self):
         return [
             re_path(
                 r"^(?P<product_id>.+)/generate-certificates/(?P<course_code>.+)/$",
-                self.admin_site.admin_view(self.generate_certificates_for_course),
+                self.admin_site.admin_view(self.issue_certificates_for_course),
                 name=ACTION_NAME_GENERATE_CERTIFICATES,
             )
         ] + url_patterns
 
     @takes_instance_or_queryset
-    def generate_certificates(self, request, queryset):  # pylint: disable=no-self-use
+    def issue_certificates(self, request, queryset):  # pylint: disable=no-self-use
         """
         Custom action to generate certificates for a collection of products
         passed as a queryset
         """
-        certificate_generated_count = helpers.generate_certificates_for_orders(
+        certificate_generated_count = helpers.issue_certificates_for_orders(
```

Comment:
```suggestion
        certificate_issued_count = helpers.issue_certificates_for_orders(
```

Reply:
actually it should be "issued_certificate_count"